Gemfile.lock

```diff
@@ -1,72 +1,76 @@
 PATH
   remote: .
   specs:
-    ruby-prof-rails (0.1.0)
+    ruby-prof-rails (1.0.0)
```

Comment:
Bumping major version may not be necessary after all. I tested my fork on our main application and it works fine.